Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Set base image (host OS)
+FROM python:3.10-slim
+
+# Set the working directory in the container
+WORKDIR /code
+
+# Set csv_files directory as mounting volume
```

Comment:
Maybe just remove them if it is no longer necessary

Reply:
Removed CSV references in Docker file, however, Docker file does not properly start python script, could you please have a look at the changes of the Dockerfile in the new commit?